css/at-rules/supports.json

```diff
@@ -80,13 +80,13 @@
             "description": "<code>selector()</code>",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": "83"
               },
               "chrome_android": {
-                "version_added": false
+                "version_added": "83"
               },
               "edge": {
-                "version_added": false
+                "version_added": "82"
```

Comment:
Do we know if Edge 82 will be released?

Reply:
Well, it’s currently listed in `edge.json`.

We can always bump this to `83` if **Microsoft** also skips **Chromium 82**.

Reply:
It looks like they've already announced. https://mspoweruser.com/microsoft-cancelling-the-version-82-release/